forge-gui/res/cardsfolder/b/brotherhoods_end.txt

```diff
@@ -4,4 +4,4 @@ Types:Sorcery
 A:SP$ Charm | Choices$ DBDamage,DBDestroy | CharmNum$ 1
 SVar:DBDamage:DB$ DamageAll | ValidCards$ Creature,Planeswalker | NumDmg$ 3 | StackDescription$ SpellDescription | SpellDescription$ CARDNAME deals 3 damage to each creature and each planeswalker.
 SVar:DBDestroy:DB$ DestroyAll | ValidCards$ Artifact.cmcLE3 | SpellDescription$ Destroy all artifacts with mana value 3 or less.
-Oracle:Choose one -\n•Brotherhood's End deals 3 damage to each creature and each planeswalker.\n•Destroy all artifacts with mana value 3 or less.
+Oracle:Choose one -\n•  Brotherhood's End deals 3 damage to each creature and each planeswalker.\n•Destroy all artifacts with mana value 3 or less.
```

Comment:
@tool4ever is that change right?

Reply:
oh yea that seems to be one space too much / not at the other mode
I'll get it changed

Reply:
I mean do we need a space after the dot?
The second one doesn't have it either

Reply:
that seems to be how Oracle does it
e. g. _Rakdos Charm_

Reply:
Well, I came to that change through finding, while searching the scripts, that it's far more common for the "newline escape+bullet character" combo to have whitespace after it (1274 hits for it against 21 hits for the "no whitespace" option). 
And, FWIW, that's what happens on Scryfall too (see [Brotherhood's End](https://scryfall.com/card/bro/128/brotherhoods-end)).
![image](https://github.com/Card-Forge/forge/assets/45150760/e980e150-83ec-4454-9bbb-79431b0e8780)

Reply:
Oops. Didn't notice the double space on one and that I missed the second `\n•` instance. I'll be more vigilant about that in the future then.